lib/components/formik/BlockNavigation.js

```diff
@@ -3,10 +3,10 @@ import Modal from "../Modal";
 import { useFormikContext } from "formik";
 import { useNavPrompt } from "react-router-nav-prompt";
 
-export default function NavigationHandler() {
+export default function NavigationHandler({ isDirty = false }) {
   const { dirty } = useFormikContext();
```

Comment:
@vyshnavrajs2019 _a, I have concern over here. We are going to use this component for non-formik forms, `useFormikContext()` function would not return a context. Since we need to know what the function is return.
If it returns `null` or `undefined`, then the component breaks.
Please test the changes in neetochat by changing version id to `git-branch` in package.json for neetoui.